### PR TITLE
Fix ruby terminating

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: src/binary/CMakeLists.txt [hunk 1]
```diff
@@ -99,7 +99,10 @@ target_link_libraries( builtinx ${YAST_YCP_LIBRARY} )
 target_link_libraries( builtinx ${YAST_PLUGIN_SCR_LIBRARY} )
 target_link_libraries( builtinx ${YAST_PLUGIN_WFM_LIBRARY} )
 target_link_libraries( builtinx ${YAST_PLUGIN_UI_LIBRARY} )
-target_link_libraries( builtinx crypt )
+find_library( CRYPT_LIBRARY crypt )
+if ( CRYPT_LIBRARY )
+  target_link_libraries( builtinx ${CRYPT_LIBRARY} )
+endif()
```
Comment (mvidner) [inline src/binary/CMakeLists.txt]: wouldn’t it be better to fail if the library is not found?

**Comment by mvidner** [comment]
The code looks good. Have you tested it with the profiler enabled?

**Comment by jreidinger** [comment]
Now tested and works as expected. Without any issues.

File: src/binary/CMakeLists.txt (hunk 1 above)
Comment (jreidinger) [inline src/binary/CMakeLists.txt]: to be honest, I try to touch cmake as less as possible. So I just tried to fix warning. I can revert it.

**Comment by mvidner** [comment]
LGTM

**Pull request merged**